src/containers/Nav.js

```diff
@@ -25,12 +25,12 @@ class Nav extends Component {
   }
 
   getTabs() {
-    if (this.props.schemas.schema) {
+    if (this.props.schemas.schema && this.props.schemas.ui && this.props.schemas.ui.entities) {
```

Comment:
I assume this is because of changes on the Security service, correct?

Reply:
Yes, this makes sure that there are UI configurations according to the changes in the security service, before rendering the dynamic part of the navigation part.

Reply:
Makes sense, thank you!